packages/hash/frontend/src/pages/[account-slug]/types/entity-type/use-entity-type.tsx

```diff
@@ -35,19 +35,23 @@ export const useEntityType = (
     entityTypeRef.current = null;
 
     void aggregateEntityTypes({ data: {} }).then(async (res) => {
```

Comment:
cc: @nathggns, just want to check you're happy with these specific changes

Reply:
Yeah I'm happy with this, thanks @Alfred-Mountfield